packages/backend-defaults/src/PackageDiscoveryService.ts

```diff
@@ -175,6 +175,6 @@ export class PackageDiscoveryService {
       }
     }
 
-    return { features };
+    return { features: Array.from(new Set(features)) };
```

Comment:
Would it also work if we instead changed the type of the "features" variable to `Set<BackendFeature>`?

Reply:
I also think it would be useful to have a warning saying that a duplicate was found and ignored, to somehow notify that this happened.

Reply:
I think this is fine as is tbh, it's a nice and simple way to dedupe with strict equality comparisons. Don't think we really need a warning either, it wouldn't be particularly actionable most of the time and is very low risk to do this deduplication.